src/aws-secrets.js

```diff
@@ -49,19 +49,40 @@ async function loadAWSSecrets(functionName) {
   }
 }
 
-async function getAWSSecrets(functionName) {
+async function getAWSSecrets(functionName, expiration) {
   const now = Date.now();
-  if (!cache.data || now > cache.expiration) {
+  if (!cache.data || now > cache.date + expiration) {
     const params = await loadAWSSecrets(functionName);
     const nower = Date.now();
     // eslint-disable-next-line no-console
     console.info(`loaded ${Object.entries(params).length} package parameter in ${nower - now}ms`);
-    if (params) {
-      cache.data = params;
-      cache.expiration = nower + CACHE_EXPIRATION;
```

Comment:
hm, `nower` is in milliseconds, but `CACHE_EXPIRATION` in seconds? Is there a factor 1000 missing?

Reply:
oh....good catch!